driver/src/main/java/org/kaazing/k3po/driver/internal/Robot.java

```diff
@@ -358,8 +358,12 @@ private void stopConfiguration() throws Exception {
                 server.setPipelineFactory(pipelineFactory(pipeline(closeOnExceptionHandler)));
             }
             for (ClientBootstrapResolver clientResolver : configuration.getClientResolvers()) {
-                ClientBootstrap client = clientResolver.resolve();
-                client.setPipelineFactory(pipelineFactory(pipeline(closeOnExceptionHandler)));
+                try {
+                    ClientBootstrap client = clientResolver.resolve();
+                    client.setPipelineFactory(pipelineFactory(pipeline(closeOnExceptionHandler)));
+                } catch (RuntimeException e) {
```

Comment:
why does this throw an exception ? Also, do you require the same above for server resolvers ?